pallets/api/src/fungibles/mod.rs

```diff
@@ -150,11 +193,9 @@ pub mod pallet {
 			spender: AccountIdOf<T>,
 			value: BalanceOf<T>,
 		) -> DispatchResultWithPostInfo {
-			let who = ensure_signed(origin.clone())
+			let owner = ensure_signed(origin.clone())
 				.map_err(|e| e.with_weight(Self::weight_approve(0, 0)))?;
-			let current_allowance = AssetsOf::<T>::allowance(id.clone(), &who, &spender);
-			let spender = T::Lookup::unlookup(spender);
-			let id: AssetIdParameterOf<T> = id.into();
+			let current_allowance = AssetsOf::<T>::allowance(id.clone(), &owner, &spender);
 
 			let return_weight = match value.cmp(&current_allowance) {
```

Comment:
This variable is called `weight` in `decrease_allowance` - suggest using the same name for consistency.

Reply:
see [refactor: fungible pallet](https://github.com/r0gue-io/pop-node/pull/153/commits/3b3ae9fcf4de6115bc5a2ecb9a92a2ca2e0d0068)